packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -96,7 +96,7 @@ class Paginator extends React.Component {
       pageButtons.push((
         <a
           aria-label={`Page ${val}`}
-          aria-current={val === selectedPage && 'page'}
+          aria-current={val === selectedPage && true}
```

Comment:
Is it possible to use just the equivalence statement Derek's referenced here? https://github.com/cerner/terra-core/pull/1850#discussion_r217542174